feature/Add-Vanity-Domains (#32)
digitonic/passona-sdk: docs/senders.md

Senders must be between **3-11** characters in length, if it is outwith this limit it will be rejected. 
New senders are subject to moderation from Digitonic.

> All teams a default sender of `88440` and a 11 character version of their team name which can be used.

Reviewer: ```suggestion
All teams get a default sender of `88440` and a 11 character version of their team name.
```
